src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/SendBuildToDataTheoremPublisher.java

```diff
@@ -409,6 +434,18 @@ public FormValidation doCheckmappingFileToUpload(
             return FormValidation.ok();
         }
 
+        public FormValidation doCheckReleaseType(
+                @QueryParameter(value = "releaseType") String releaseType
+        )
+        {
+            if (releaseType.isEmpty())
```

Comment:
In that context, does that mean that the field will be set, but as an empty string, or that the field won't be set at all in the multipart form? Only the second is valid.

Reply:
No, it will just warn the customer when he configures the action but will keep the value ( soft locking)
When the customer executes the action if the value is empty it won't be set
If the value differs from the list the action will raise an exception explaining that the type is not supported